generated/src/aws-cpp-sdk-s3-crt/source/model/AbortMultipartUploadRequest.cpp

```diff
@@ -27,6 +26,28 @@ AbortMultipartUploadRequest::AbortMultipartUploadRequest() :
 {
 }
 
+bool AbortMultipartUploadRequest::HasEmbeddedError(Aws::IOStream &body,
+  const Aws::Http::HeaderValueCollection &header) const
+{
+  // Header is unused
+  (void) header;
```

Comment:
nit but please change: `AWS_UNREFERENCED_PARAM(header);` or just remove the header variable name from the signature.

Reply:
ack